spec/lib/api/1.1/skus-spec.js

```diff
@@ -16,7 +17,9 @@ describe('Http.Api.Skus', function () {
         ]);
         return helper.startServer([
             dihelper.simpleWrapper(workflowApiService, 'Http.Services.Api.Workflows')
-        ]);
+        ]).then(function() {
+            _ = helper.injector.get('_');
```

Comment:
As a CC you want me to trace through the code and out of this repository to find that you inject this as a global variable somewhere else?  You feel so strongly about it that you nitpicked this line of code?